src/templates/article.js

```diff
@@ -46,6 +49,26 @@ const ArticlesTemplate = (props) => {
         isMounted && window.scrollTo(0, 0)
     }, [isMounted])
 
+    const barElement = useRef(null)
+    const scrollElement = useRef(null)
+
+    const scrollFunc = () => {
+        const winScroll = document.body.scrollTop || document.documentElement.scrollTop
+        const height = document.documentElement.scrollHeight - document.documentElement.clientHeight
+        const scrolled = (winScroll / height) * 130
+        barElement.current.style.width = scrolled + '%'
+    }
+
+    useEffect(() => {
+        const windowScroll = () => {
+            window.addEventListener('scroll', scrollFunc)
+        }
+        windowScroll()
```

Comment:
```suggestion
        window.addEventListener('scroll', scrollFunc)
```

Reply:
Since we're using it right away and only once here. We don't need to declare it as a named function.